src/global.d.ts

```diff
@@ -0,0 +1,7 @@
+import { JSDOM } from 'jsdom';
+
+declare global {
```

Comment:
I dont think this should be in SRC as its about the test

Reply:
without this jsdom won't resolve iif put inside test folder it wont do anything